openstack/ces/v1/alarms/delete_alarm.go

```diff
@@ -0,0 +1,11 @@
+package alarms
+
+import golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+
+func DeleteAlarm(client *golangsdk.ServiceClient, id string) (err error) {
+	// DELETE /V1.0/{project_id}/alarms/{alarm_id}
+	_, err = client.Delete(client.ServiceURL("alarms", id), &golangsdk.RequestOpts{
+		OkCodes: []int{204},
```

Comment:
As far as I remember, for DELETE 204 is default OK code, so you can skip request opts definition here